### ISSUE TITLE
Word wrap alignment

### ISSUE THREAD

**Issue by s-fong** [opencor/opencor#1680]
For continuation of text which begins on a second line, make this second line have the same alignment as the start of the first line.

**Comment by agarny** [comment]
Fair point, I will have a look whether this can actually be done with the editing widget we are using.
